arraysum.c

```diff
@@ -0,0 +1,46 @@
+//Olivier Gabison - When inputting the numbers in the array, finds the sum of all the other integers in the array and stores it in that
+//value.
+
+#include <stdio.h>
```

Comment:
Does what it needs to, but only for arrays with five elements. user prompt instructions should have been clear about that.